opentracing-spring-jaeger-starter/src/main/java/io/opentracing/contrib/java/spring/jaeger/starter/JaegerConfigurationProperties.java

```diff
@@ -13,10 +13,18 @@
  */
 package io.opentracing.contrib.java.spring.jaeger.starter;
 
+import java.util.HashMap;
+import java.util.Map;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+
 @ConfigurationProperties("opentracing.jaeger")
 public class JaegerConfigurationProperties {
+  /**
+   * Prefix for all properties used to configure the Jaeger tracer.
+   */
+  static final String JAEGER_PREFIX = "JAEGER_";
+  static final String JAEGER_TAGS = JAEGER_PREFIX + "TAGS";
```

Comment:
It can be consumed from jaeger-core - Configuration.JAEGER_TAGS

Reply:
Fixed